src/pages/studyView/table/MultiSelectionTable.tsx

```diff
@@ -324,7 +324,11 @@ export class MultiSelectionTable extends React.Component<
                         data.numberOfAlteredCases,
                         data.matchingGenePanelIds || [],
                         this.toggleModal,
-                        { marginLeft: cellMargin }
+                        {
```

Comment:
@aditygrg2 is this still necessary now that we have the class?

Reply:
Hi @alisman I have committed after updating the function. I was not doing it earlier as I thought the function might change some important things, sorry for not checking it earlier correctly. Now, the new commit solves this.